docker/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM nextcloud:30-apache
 
 COPY entrypoint.sh /
 
-RUN apt-get update && apt-get install -y sqlite3
+RUN apt-get update && apt-get install -y sqlite3 nano
```

Comment:
I think this is user preference, do we really want to install this?

Reply:
Not sure I just want a proper setup within the docker container to do stuff, I will drop it from this PR and check for alternatives